### ISSUE TITLE
Attributes property

### ISSUE THREAD

**Issue by digitalsadhu** [jthoms1/bookshelf-jsonapi#13]
Hey, just reading through your tests and notice that properties are not placed under an "attributes" key. Is this due to jsonapi being at an older version at the time of writing?

**Comment by jthoms1** [comment]
Correct. Mostly this package is depending on jsonapi2simple for the implementation.  That package is still using an older implemetnation of jsonapi. Probably pre 1.0.

**Comment by digitalsadhu** [comment]
Thanks for the info.
Any plans to update jsonapi2simple? If jsonapi2simple were updated would that be enough to update bookshelf-jsonapi or would it require additional changes to this repo as well?

**Comment by digitalsadhu** [comment]
Thinking about it, at the very least you would need to swap `put` out for `patch`
